regression/ansi-c/linking1/other.c

```diff
@@ -0,0 +1,14 @@
+#ifndef TEST2
+#include "foo.h"
+#else
+inline void foo(int x)
```

Comment:
Comment in either the code or the `.desc` files what you're expecting here. I might be being thick, but I don't follow what you're expecting to differ, except that the non-static inline `foo` would leave an external-linkage version of the function body around for a potential linked object expecting an `extern void foo(int)`, which doesn't in fact exist in this test?

Reply:
I have added explanations in the .desc files.

Reply:
Aha, excellent, thankyou!